cmake/detect-intrinsics.cmake

```diff
@@ -220,7 +220,7 @@ macro(check_neon_ld4_intrinsics)
         endif()
     endif()
     # Check whether compiler supports loading 4 neon vecs into a register range
-    set(CMAKE_REQUIRED_FLAGS "${NEONFLAG}")
+    set(CMAKE_REQUIRED_FLAGS "${NEONFLAG} ${NATIVEFLAG} ${ZNOLTOFLAG}")
```

Comment:
I noticed that this test lacked the NATIVEFLAG, and I don't think it should, since the -march is conditional above.

Reply:
IOW: If NativeInst is enabled, this would always fail unless `-march=native` is already part of cflags from outside of CMake.